mdal/frmts/mdal_driver.hpp

```diff
@@ -44,6 +44,9 @@ namespace MDAL
       // loads datasets
       virtual void load( const std::string &uri, Mesh *mesh, MDAL_Status *status );
 
+      // save mesh
+      virtual void save( const std::string &uri, MDAL_Status *status, Mesh *mesh );
```

Comment:
can you shuffle arguments so they are in the same order as load function pleasE?

Reply:
done